docs/how-bilt-works.md

```diff
@@ -18,17 +18,17 @@ packages were already built](#how-bilt-knows-which-packages-were-already-built))
 of the dependency graph, including only those packages it needs to build, while keeping the
 following invariants:
 
-1. A package will be built iff it is in the graph subset that links all the uptos to the froms in the
+1. A package will be built if it is in the graph subset that links all the uptos to the froms in the
```

Comment:
Actually, I did mean "iff", which in mathematics means "if and only if" (maybe we should change it to "if and only if"?)

Reply:
When I read the docs, I didn't understand the meaning of `iff`.
Do you want me to change it to `if and only if` or keep it as it was?

Reply:
You decide!